frontend/public/components/routes.tsx

```diff
@@ -512,9 +500,56 @@ export const RoutesDetailsPage: React.FC<RoutesDetailsPageProps> = (props) => (
     pages={[navFactory.details(detailsPage(RouteDetails)), navFactory.editYaml()]}
   />
 );
-export const RoutesList: React.FC = (props) => (
-  <Table {...props} aria-label="Routes" Header={RouteTableHeader} Row={RouteTableRow} virtualize />
-);
+
+export const RoutesList: React.FC = (props) => {
+  const { t } = useTranslation();
+  const RouteTableHeader = () => {
+    return [
+      {
+        title: t('network-route~Name'),
+        sortField: 'metadata.name',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[0] },
+      },
+      {
+        title: t('network-route~Namespace'),
+        sortField: 'metadata.namespace',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[1] },
+        id: 'namespace',
+      },
+      {
+        title: t('network-route~Status'),
+        props: { className: tableColumnClasses[2] },
+      },
+      {
+        title: t('network-route~Location'),
+        sortField: 'spec.host',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[3] },
+      },
+      {
+        title: t('network-route~Service'),
+        sortField: 'spec.to.name',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[4] },
+      },
+      {
+        title: '',
+        props: { className: tableColumnClasses[5] },
+      },
+    ];
+  };
+  return (
+    <Table
+      {...props}
+      aria-label="Routes"
```

Comment:
This aria label should be marked for translation.

Reply:
fixed, good catch, thought I did a search on those files, just redid one, no untranslated `aria-label`'s found in these files.

Reply:
I missed it my first pass! :) I swear they hide.